src/pandas_profiling/model/expectation_algorithms.py

```diff
@@ -36,6 +36,11 @@ def numeric_expectations(name, summary, batch, *args):
             name, strictly=summary["monotonic_decrease_strict"]
         )
 
+    if summary.get("min") or summary.get("max"):

```

Comment:
Strictly `get` should be used for obtaining the value, not for membership checks. 

(better to use `in`: `if any(k in summary for k in ["min", "max"])`)